internal/business/server/healthcheck_handler.go

```
@@ -21,6 +21,41 @@ import (
 	"github.com/openkcm/checker/internal/healthcheck"
 )
 
+func maskResponse(r *healthcheck.Response) *healthcheck.Response {
+	copy := *r
```

golangci-lint (2.12.2)

[error] variable copy has same name as predeclared identifier

(predeclared)